src/warehouses/components/WarehouseList/WarehouseList.tsx

```diff
@@ -140,9 +140,13 @@ const WarehouseList: React.FC<WarehouseListProps> = props => {
                 {maybe<React.ReactNode>(() => warehouse.name, <Skeleton />)}
               </TableCell>
               <TableCell className={classes.colZones} data-test="zones">
-                {mapEdgesToItems(warehouse?.shippingZones)
-                  .map(({ name }) => name)
-                  .join(", ") || <Skeleton />}
+                {disabled ? (
+                  <Skeleton />
+                ) : (
+                  mapEdgesToItems(warehouse?.shippingZones)
+                    ?.map(({ name }) => name)
+                    .join(", ") || "-"
+                )}
```

Comment:
Why using disabled here?

```suggestion
                {warehouse?.shippingZones === undefined ? (
                  <Skeleton />
                ) : (
                  mapEdgesToItems(warehouse?.shippingZones).map(({ name }) => name)
                    .join(", ") || "-"
                )}
```

It would be even nicer to extract `mapEdgesToItems(warehouse?.shippingZones)` to separate variable

Reply:
i used `disabled` because it's assigned the value of `loading` a few levels up, and i need it to know whether the item has finished loading. i wanted to avoid introducing another prop which would end up having the same value as `disabled`. seeing as how there seems to be an ongoing discussion on the topic elsewhere, i might end up doing it anyway.

Reply:
I'm strongly against using `disabled` as a loading state only because it's dependent on `loading` a few levels higher. `warehouse?.shippingZones === undefined` is the only case when skeleton should be displayed.